tests/test_app.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+from app.src.app import app
+
+@pytest.fixture
+def client():
+    app.config['TESTING'] = True
```

Comment:
Rejecting and requesting for changes

Reply:
@skydonline please look at my updated commits, resolved bad code